### PR TITLE
Notebuttons

### PR DESCRIPTION
**Pull request by ichang1**
Implemented tag functionality for person. There is another form in the person edit view which has one field, a tags ModelMultipleChoiceField where the user searches and selects the tags they want. 

- The available tags to choose from should be all the tags. The tags that are already tagged with the person should be already checked. 
- There is a button near the end of the person edit form that allows user to add a new tag. User is only able to create a new tag one at a time for now. When the user submits new tag, they will be redirected to the person edit page and the new tag should be an option and be already selected.
- Once a tag is unchecked, the associated tag_x will be deleted after submitting the person edit form. By default, all the tags that have a tag_x linking the person and the tag should be selected upon visiting the person edit page. 

Below are example pictures showing the changes fopr visiting the person edit page, adding a tag, then being redirected back
person_edit_before
add_tag
person_edit_after

### CODE REVIEW

File: escalate/core/views/model_tag.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def post(self, request, *args, **kwargs):
             #above url not being redirected to in super.post so I brute-forced redirect
             return redirect(f'/{model_name}/{model_pk}')
```
Comment (ichang1) [inline escalate/core/views/model_tag.py:34]: I tried changing the success url if the cancel button was clicked, but when I tested it, it doesn't work, so I improvised for now. Hoping to see if you had any ideas on why it didn't work.

Comment (ichang1) [inline escalate/core/views/model_tag.py:34]: I tired changing the success url when the cancel button is clicked, but when I tested it, the url changed, but I was redirect back to the same page. I was wondering if there are any ideas about why this happens/how to fix it.

Comment (ichang1) [inline escalate/core/views/model_tag.py:34]: Tried changing success url when the cancel button was clicked, but when submit is clicked, the user is redirected back to the add tag page not the person edit page even though the success url changed. Same problem occurs when clicking submit. Hoping there are ideas on why this happens

Comment (vshekar) [reply escalate/core/views/model_tag.py:34]: Interesting, I'll take a look at this after I accept this

**Pull request merged**